ext/bg/js/backend.js

```diff
@@ -515,7 +515,7 @@ class Backend {
 
     async _onApiAudioGetUrl({definition, source, optionsContext}) {
         const options = this.getOptions(optionsContext);
-        return await audioGetUrl(definition, source, options);
+        return await this.audioUriBuilder.getUri(source, definition, options);
```

Comment:
I think either `AudioUriBuilder.getUri` should be changed to the `definition, source, options` or the other way around, whichever is more desirable, to avoid confusion

Reply:
I'll update `source` to be the first argument consistently, since there's precedent for that in multiple places. There are a few places where this parameter is named either `source` or `mode`, and I'll try to settle on only one of those also.
https://github.com/FooSoft/yomichan/blob/0cd1f8f20a1b632847e14b280849e4af1b6dd9ad/ext/bg/js/translator.js#L144-L157
https://github.com/FooSoft/yomichan/blob/0cd1f8f20a1b632847e14b280849e4af1b6dd9ad/ext/mixed/js/display.js#L455-L477

Reply:
Ok, I guess it can be done in another PR later too if the amount of changes needed would otherwise explode. The variable names are also becoming more relevant when using object destructuring as I noticed in https://github.com/FooSoft/yomichan/commit/0112dbab33ab214f9e1dc930558833956d4ad1c4.

Reply:
I'll take care of it in this PR, just haven't gotten around to it yet.

Reply:
Actually, it's considerably more simple to do the opposite of what I said. So in the interest of minimal changes, and hopefully minimal breakage, I went that route. 0cbf427ab50061b48c9027e63e9ee8a209946d37

Reply:
Looks good, tested that it still works